### ISSUE TITLE
Minification strips whitespace for wrapped lines

### ISSUE THREAD

**Issue by chrisvoll** [styled-components/babel-plugin-styled-components#69]
### Context

[Prettier 1.5.0](https://github.com/prettier/prettier/releases/tag/1.5.0) introduces support for formatting styled-components output. One of the changes it makes is wrapping long lines to the next line.

The minification setting in `babel-plugin-styled-components` strips out this whitespace entirely.

### Test Case

Input `test.js`:

```javascript
import styled from 'styled-components';

const output = styled.li`
  animation: really-really-really-really-really-really-long-name 2s linear infinite;
`;

console.log(output.componentStyle.rules);
```

```
$ jest test.js
[ 'animation: really-really-really-really-really-really-long-name 2s linear infinite;' ]
```

Output from `prettier test.js`, note how there isn't a space between `linear` and `infinite`:

```javascript
import styled from 'styled-components';

const output = styled.li`
  animation: really-really-really-really-really-really-long-name 2s linear
    infinite;
`;

console.log(output.componentStyle.rules);
```

```
$ jest test.js
[ 'animation: really-really-really-really-really-really-long-name 2s linearinfinite;' ]
```

With minification turned off, the whitespace is preserved:

```
$ jest test.js
[ '\n  animation: really-really-really-really-really-really-long-name 2s linear\n    infinite;\n' ]
```

**Comment by philpl** [comment]
Oh, jeeeeez. Thanks for spotting this edge case :smile: I'll see what I can do

**Comment by chrisvoll** [comment]
thanks @philpl! Let me know if there's anything else I can help with!

**Comment by philpl** [comment]
@chrisvoll This PR should fix it. Wanna take a look whether it works for you? https://github.com/styled-components/babel-plugin-styled-components/pull/70

**Comment by philpl** [comment]
Published in 1.1.6
